ts/features/pn/utils/index.ts

```diff
@@ -54,10 +54,8 @@ export const extractPNOptInMessageInfoIfAvailable = (
     O.fromNullable,
     O.chain(serviceId =>
       pipe(
-        state.backendStatus.status,
-        O.map(
-          backendStatus => backendStatus.config.pn.optInServiceId === serviceId
-        )
+        state.remoteConfig,
+        O.map(c => c.pn.optInServiceId === serviceId)
```

Comment:
Should we use `remoteConfig` instead of `c`, to improve readability?

Reply:
```suggestion
        O.map(remoteConfig => remoteConfig.pn.optInServiceId === serviceId)
```